analyses/cell-type-neuroblastoma-04/scripts/01_train-singler-model.R

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env Rscript
+#
+# This script trains a SingleR model from a given NBAtlas object
+# The script additionally requires an input SCE object to determine genes to restrict to (TODO: make this a TSV?)
```

Comment:
One idea is you could just pull our tx2gene file from `scpca-references` and read that in to get the gene id list to restrict too rather than reading in an SCE? But I think either one is fine.